src/api/json/catalog.json

```diff
@@ -226,6 +226,16 @@
         "bitbucket-pipelines.yml"
       ]
     },
+    {
```

Comment:
can we somehow put step.yml into this list also? I would only change the name and the root ref to be `StepModel` instead of `BitriseDataModel `

Reply:
That would result in a different tree (as it doesn not _contain_ the `BitriseDataModel`), not sure how we could merge the two. What did you have in mind exactly?

Reply:
I'll add a schema for `bitrise-step` as well